index.js

```diff
@@ -0,0 +1,21 @@
+$(document).delegate('a:Event(click)', 'click', function (e) { e.preventDefault(); });
+
+$( ".readmore" ).click(function() {
+  $( "#show-this-on-click" ).slideDown();
+});
+$( ".readmore" ).click(function() {
+  $( "#hidebtn" ).show();
+});
+$( ".readmore" ).click(function() {
+  $( ".readmore" ).hide();
+});
+
```

Comment:
instead of creating 3 click listeners, you only need to click one and then put as many lines of code inside as you want 

```
$( ".readmore" ).click(function() {
  $( "#show-this-on-click" ).slideDown();
  $( "#hidebtn" ).show();
  $( ".readmore" ).hide();
});
```